api/helpers.go

```diff
@@ -4,22 +4,13 @@ import (
 	"encoding/json"
 	"net/http"
 
-	jwt "github.com/dgrijalva/jwt-go"
-	"github.com/netlify/gocommerce/claims"
+	"github.com/Sirupsen/logrus"
 )
 
 func sendJSON(w http.ResponseWriter, status int, obj interface{}) {
 	w.Header().Set("Content-Type", "application/json")
 	w.WriteHeader(status)
-	encoder := json.NewEncoder(w)
-	encoder.Encode(obj)
-}
-
-func userIDFromToken(token *jwt.Token) string {
-	if token == nil {
-		return ""
+	if err := json.NewEncoder(w).Encode(obj); err != nil {
```

Comment:
I am betting they're close enough that it doesn't truly matter that much. You could just toss together a quick benchmark if you were feeling really proactive. I think regardless of the performance we should consider doing the "right" thing.